src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -13,6 +13,7 @@
     <CLSCompliant>false</CLSCompliant>
     <AssemblyVersion>1.2.0.0</AssemblyVersion>
     <PackageTargetFramework>dotnet5.2</PackageTargetFramework>
+    <LangVersion>6</LangVersion>
```

Comment:
We should definitely not do this for only this project. We have to wait until all of them can switch.